### PR TITLE
feat: add memo tag information section in LLD recieve flow

### PR DESCRIPTION
**Pull request by themooneer**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.**  (covered by integration tests)
- [x] **Impact of the changes:** 
  - add a div section to Receive flow step 2 and 3 , it should be covered by e2e tests

### 📝 Description

Simply add an alert section to inform user about MemoTag

https://github.com/user-attachments/assets/574c9067-da29-45a3-ae09-f870979fe58c

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-9632

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/newArch/features/MemoTag/__tests__/MemoTagInfoBody.test.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  it("should display MemoTagInfoBody correctly", async () => {
+    render(<MemoTagInfoBody />);
+
+    const memoTagInfoBody = screen.getByTestId("memo-tag-info-body");
+    const learnMoreLink = screen.getByText(/Learn more about Tag\/Memo/);
+
+    expect(memoTagInfoBody).toBeVisible();
+    expect(learnMoreLink).toBeVisible();
+
+    fireEvent.click(learnMoreLink);
```
Comment (LucasWerey) [inline apps/ledger-live-desktop/src/newArch/features/MemoTag/__tests__/MemoTagInfoBody.test.tsx:24]: for this you can do {user}= render(<MemoTagInfoBody />); at the init of the test
then user.click

File: apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx [hunk 2]
```diff
@@ -103,6 +104,11 @@ const Receive1ShareAddress = ({
           <UTXOAddressAlert />
         </Box>
       )}
+      <FeatureToggle featureId="lldMemoTag">
```
Comment (LucasWerey) [inline apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx:107]: you can use the feature flag selector for this one

File: apps/ledger-live-desktop/src/newArch/features/MemoTag/__tests__/MemoTagInfoBody.test.tsx (hunk 1 above)
Comment (themooneer) [reply apps/ledger-live-desktop/src/newArch/features/MemoTag/__tests__/MemoTagInfoBody.test.tsx:24]: I realy prefer to rely on native jest/rtl  capability for unit testing component behaviour

File: apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx (hunk 2 above)
Comment (themooneer) [reply apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx:107]: answered above

Comment (themooneer) [reply apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx:107]: the wrapper already use it

Comment (thesan) [inline apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepReceiveFunds.tsx:107]: Kevin and Anthony suggested to only show the memos/tags message for coins that might need them. So I added a list in LLD: https://github.com/LedgerHQ/ledger-live/pull/7989/files#diff-95f10ef90834180737095894a09612d65e36dc5ec286fb07a2ed743b3cede50e

I'm a bit worried that we might miss some of them (especially on the send flow). What do you think of this ?

**Review by thesan — APPROVED** [review]
LGTM!

**Pull request merged**